raiden/raiden_service.py

```diff
@@ -631,7 +631,7 @@ def _initialize_payment_statuses(self, chain_state: ChainState):
                 balance_proof = transfer.balance_proof
                 self.targets_to_identifiers_to_statuses[target][identifier] = PaymentStatus(
                     payment_identifier=identifier,
-                    amount=balance_proof.transferred_amount,
+                    amount=balance_proof.locked_amount,
```

Comment:
Come on, this is the wrong value **again**

Reply:
ARGH ffs. It should be the locked amount of this transfer, not the overall locked amount. Is that what you mean? Damn even wrote a test ...

Reply:
> locked amount of this transfer, not the overall locked amount

yes

Reply:
Apologies. My test gave me false sense of security since it failed before but worked after the change. But using the locked amount will fail the minute multiple transfers are routed through the channel. Let me make an amending PR for this.

Reply:
wait, now i am lost... should this be `transfer.lock.amount`?

Reply:
yes .. I am editing the test now to make it fail with my implementation too so that we are sure no regressions are inserted in the future

Reply:
New PR with proper fix and to also have the test create a scenario where the channel has additional locked amount unrelated to the transfer: https://github.com/raiden-network/raiden/pull/3490/